tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRURLS.java

```diff
@@ -88,6 +89,50 @@
      */
     public static final String UNKNOWN_REVISION = "(unknown)";
 
+    public static final String GENERAL_HELP_URL = "https://cldr.unicode.org/translation/";
+    public static final String ADMIN_HELP_URL = "https://cldr.unicode.org/index/survey-tool/survey-tool-administration";
+    public static final String CAPITALIZATION_URL = "https://cldr.unicode.org/translation/translation-guide-general/capitalization";
+    public static final String CHARACTERS_HELP = "https://cldr.unicode.org/translation/characters";
```

Comment:
Maybe some factoring? 
```suggestion
    public static final String CHARACTERS_HELP = GENERAL_HELP_URL + "characters";
```

Reply:
It's worth considering building them up from pieces, as you suggest. However I think it might be better to keep the complete URL strings in the source code so they're _searchable_. Whenever someone discovers a broken link, it will be easy to find the URL in the source and fix it, without needing to remember where to look (CLDRURLS.java). The string "characters" might occur in a zillion source files. Finding CHARACTERS_HELP by way of GENERAL_HELP_URL could be time-consuming and error-prone.